spring-cloud-dataflow-server/docker-compose-prometheus-rsocket.yml

```diff
@@ -0,0 +1,59 @@
+version: '3'
+
+services:
+
+  dataflow-server:
+    image: springcloud/spring-cloud-dataflow-server:${DATAFLOW_VERSION:?DATAFLOW_VERSION is not set!}
+    container_name: dataflow-server
+    volumes:
+      - '/tmp/apps:/apps/'
+    ports:
+      - "9393:9393"
+    environment:
+      - spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.kafka.binder.brokers=PLAINTEXT://kafka:9092
+      - spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.kafka.streams.binder.brokers=PLAINTEXT://kafka:9092
+      - spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.kafka.binder.zkNodes=zookeeper:2181
+      - spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.kafka.streams.binder.zkNodes=zookeeper:2181
+      - spring.cloud.skipper.client.serverUri=http://skipper-server:7577/api
+      - spring.cloud.dataflow.applicationProperties.stream.management.metrics.export.prometheus.enabled=true
```

Comment:
For consistency, it'd be nice if we could embed a JSON document here similar to what we have done for all the manifest files for CF. See: https://github.com/spring-io/dataflow.spring.io/pull/117/files#diff-20356809c25742a64d85361aae15b399R187.

It'd be so much better read a formatted documented as opposed to flat env-vars.

Reply:
will give it a try